FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -15,8 +15,18 @@ servers:
 
 tags:
 - name: System
-  description: High level system and service information
-- name: Fault
+  description: High level system and service information. These are deprecated and replaced by the component endpoints to avoid confusion about ResilienX FRAIHMWORK terminology.
+  
+- name: Components
+  description: All component related api endpoints
+  
+- name: Faults
+  description: All fault related api endpoints
+  
+- name: Component Updates
+  description: High level component  information
```

Comment:
What is the difference between Components and Component Updates?

Reply:
Not too much. This was a hold over from merging - I've gotten rid of the extra tags to avoid confusion.